images/win/scripts/Installers/Install-AzureDevSpacesCLI.ps1

```diff
@@ -0,0 +1,10 @@
+################################################################################
+##  File:  Install-AzureDevSpacesCLI.ps1
+##  Desc:  Install Azure Dev Spaces CLI
+################################################################################
+
+# Install Azure Dev Spaces CLI
+Install-Binary -Url "https://aka.ms/get-azds-windows" -Name "Azure Dev Spaces CLI.exe" -ArgumentList ("/quiet")
+$env:Path = $env:Path + ";C:\Program Files\Microsoft SDKs\Azure\Azure Dev Spaces CLI"

```

Comment:
Please pay attention that this line will update Path only for this single script. If we need to add this folder to PATH permanently, we should use https://github.com/actions/virtual-environments/blob/main/images/win/scripts/ImageHelpers/PathHelpers.ps1#L30